extension/data/tbcore.js

```diff
@@ -1412,6 +1411,14 @@ export async function getToolboxDevs () {
         // TBStorage.setSetting('Notifier', 'lastSeenModmail', now); // don't spam 100 new mod mails on first install.
         // TBStorage.setSetting('Notifier', 'modmailCount', 0);
         TBStorage.setSetting(SETTINGS_NAME, 'debugMode', false);
+    }
+
+    // First run changes for major and minor releases only
+    // https://semver.org
+    const shortVersionMinor = Math.floor(shortVersion / 1e2);
+    const lastVersionMinor = Math.floor(lastVersion / 1e2);
```

Comment:
I don't feel like `1e2` is any clearer than `100` here...
```suggestion
    const shortVersionMinor = Math.floor(shortVersion / 100);
    const lastVersionMinor = Math.floor(lastVersion / 100);
```